ballerina/tests/test_services.bal

```diff
@@ -1185,4 +1187,14 @@ service /subscriptions on basicListener {
         }
         return filteredArray.toStream();
     }
+
+    isolated resource function subscribe values() returns stream<int> {
+        int[] array = [];
+        int _ = array.remove(0);
```

Comment:
Any reason to include this line? In which test case, this resource function is used?

Reply:
It was there to return an unexpected error from a subscription resource function. So the `notifyFailure()` method in `SubscriptionCallback` class can be invoked. So that it can be covered by a test. 
I used it in `testSubscriptionFunctionWithErrors` test case